### ISSUE TITLE
Compiling on android fails to build luajit

### ISSUE THREAD

**Issue by ErikasKontenis** [opentibiabr/otclient#383]
### Priority

Low

### Area

- [ ] Data
- [X] Source
- [ ] Docker
- [ ] Other

### What happened?

I am compiling on windows based on the https://github.com/mehah/otclient/wiki/Compiling-on-Android page. However the compiling fails to build luajit on all android triplets. I have created the issue regarding this community triplet build failure logs: https://github.com/microsoft/vcpkg/issues/28724

But is there anything I can do about that? Does the code builds for anyone? I tried to rollback vcpkg to previous version similar to wiki page edit date of the month november but it still failed to build.

Any help would be greatly appreciated! :) 

### What OS are you seeing the problem on?

Windows

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tuliomagalhaes** [comment]
Hello! Did you download the Android Libraries described in the beginning of the tutorial and put it inside the folder described on **Step 4**?

**Comment by ErikasKontenis** [comment]
Hello, @tuliomagalhaes! Thank you for reply! I still didn't managed to solve this issue on my own yet. Yes I did not forget this step. And I have here lib & include folder just near the put_libs_here.txt file.

**Comment by tuliomagalhaes** [comment]
Can you please upload the logs somewhere?

**Comment by ErikasKontenis** [comment]
I don't have much more than I provided in the https://github.com/microsoft/vcpkg/issues/28724. I can show you that all 4 build tasks are failing due the same reason cannot build LuaJIT:
<details>
  <summary>Android build log</summary>
Executing tasks: [:app:assembleRelease] in project C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android

Starting Gradle Daemon...
Gradle Daemon started in 2 s 688 ms

> Configure project :app
AGPBI: {"kind":"warning","text":"The option setting 'android.prefabVersion=2.0.0' is experimental.","sources":[{}]}

> Task :app:buildKotlinToolingMetadata UP-TO-DATE
> Task :app:preBuild UP-TO-DATE
> Task :app:preReleaseBuild UP-TO-DATE
> Task :app:compileReleaseAidl NO-SOURCE
> Task :app:compileReleaseRenderscript NO-SOURCE
> Task :app:dataBindingMergeDependencyArtifactsRelease UP-TO-DATE
> Task :app:dataBindingMergeGenClassesRelease UP-TO-DATE
> Task :app:generateReleaseResValues UP-TO-DATE
> Task :app:generateReleaseResources UP-TO-DATE
> Task :app:mergeReleaseResources UP-TO-DATE
> Task :app:packageReleaseResources UP-TO-DATE
> Task :app:parseReleaseLocalResources UP-TO-DATE
> Task :app:dataBindingGenBaseClassesRelease UP-TO-DATE
> Task :app:generateReleaseBuildConfig UP-TO-DATE
> Task :app:checkReleaseAarMetadata UP-TO-DATE
> Task :app:mapReleaseSourceSetPaths UP-TO-DATE
> Task :app:createReleaseCompatibleScreenManifests UP-TO-DATE
> Task :app:extractDeepLinksRelease UP-TO-DATE
> Task :app:processReleaseMainManifest UP-TO-DATE
> Task :app:processReleaseManifest UP-TO-DATE
> Task :app:processReleaseManifestForPackage UP-TO-DATE
> Task :app:processReleaseResources UP-TO-DATE
> Task :app:compileReleaseKotlin UP-TO-DATE
> Task :app:javaPreCompileRelease UP-TO-DATE
> Task :app:compileReleaseJavaWithJavac UP-TO-DATE
> Task :app:extractProguardFiles UP-TO-DATE
> Task :app:lintVitalAnalyzeRelease UP-TO-DATE
> Task :app:lintVitalReportRelease UP-TO-DATE
> Task :app:lintVitalRelease

> Task :app:configureCMakeRelWithDebInfo[arm64-v8a]
C/C++: release|arm64-v8a :vcpkg_android.cmake: VCPKG_TARGET_TRIPLET was set to arm64-android
C/C++: release|arm64-v8a :vcpkg_android.cmake: CMAKE_TOOLCHAIN_FILE was set to C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg/scripts/buildsystems/vcpkg.cmake
C/C++: release|arm64-v8a :vcpkg_android.cmake: VCPKG_CHAINLOAD_TOOLCHAIN_FILE was set to C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393/build/cmake/android.toolchain.cmake
C/C++: release|arm64-v8a :CMake Error at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/scripts/buildsystems/vcpkg.cmake:883 (message):
C/C++: release|arm64-v8a :  vcpkg install failed.  See logs for more information:
C/C++: release|arm64-v8a :  C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\arm64-v8a\vcpkg-manifest-install.log
C/C++: release|arm64-v8a :Call Stack (most recent call first):
C/C++: release|arm64-v8a :  C:/Users/erika/AppData/Local/Android/Sdk/cmake/3.22.1/share/cmake-3.22/Modules/CMakeDetermineSystem.cmake:124 (include)
C/C++: release|arm64-v8a :  CMakeLists.txt:32 (project)

> Task :app:configureCMakeRelWithDebInfo[arm64-v8a] FAILED

> Task :app:configureCMakeRelWithDebInfo[armeabi-v7a]
C/C++: release|armeabi-v7a :vcpkg_android.cmake: VCPKG_TARGET_TRIPLET was set to arm-neon-android
C/C++: release|armeabi-v7a :vcpkg_android.cmake: CMAKE_TOOLCHAIN_FILE was set to C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg/scripts/buildsystems/vcpkg.cmake
C/C++: release|armeabi-v7a :vcpkg_android.cmake: VCPKG_CHAINLOAD_TOOLCHAIN_FILE was set to C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393/build/cmake/android.toolchain.cmake
C/C++: release|armeabi-v7a :CMake Error at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/scripts/buildsystems/vcpkg.cmake:883 (message):
C/C++: release|armeabi-v7a :  vcpkg install failed.  See logs for more information:
C/C++: release|armeabi-v7a :  C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\armeabi-v7a\vcpkg-manifest-install.log
C/C++: release|armeabi-v7a :Call Stack (most recent call first):
C/C++: release|armeabi-v7a :  C:/Users/erika/AppData/Local/Android/Sdk/cmake/3.22.1/share/cmake-3.22/Modules/CMakeDetermineSystem.cmake:124 (include)
C/C++: release|armeabi-v7a :  CMakeLists.txt:32 (project)

> Task :app:configureCMakeRelWithDebInfo[armeabi-v7a] FAILED

> Task :app:configureCMakeRelWithDebInfo[x86]
C/C++: release|x86 :vcpkg_android.cmake: VCPKG_TARGET_TRIPLET was set to x86-android
C/C++: release|x86 :vcpkg_android.cmake: CMAKE_TOOLCHAIN_FILE was set to C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg/scripts/buildsystems/vcpkg.cmake
C/C++: release|x86 :vcpkg_android.cmake: VCPKG_CHAINLOAD_TOOLCHAIN_FILE was set to C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393/build/cmake/android.toolchain.cmake
C/C++: release|x86 :CMake Error at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/scripts/buildsystems/vcpkg.cmake:883 (message):
C/C++: release|x86 :  vcpkg install failed.  See logs for more information:
C/C++: release|x86 :  C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\x86\vcpkg-manifest-install.log
C/C++: release|x86 :Call Stack (most recent call first):
C/C++: release|x86 :  C:/Users/erika/AppData/Local/Android/Sdk/cmake/3.22.1/share/cmake-3.22/Modules/CMakeDetermineSystem.cmake:124 (include)
C/C++: release|x86 :  CMakeLists.txt:32 (project)

> Task :app:configureCMakeRelWithDebInfo[x86] FAILED

> Task :app:configureCMakeRelWithDebInfo[x86_64]
C/C++: release|x86_64 :vcpkg_android.cmake: VCPKG_TARGET_TRIPLET was set to x64-android
C/C++: release|x86_64 :vcpkg_android.cmake: CMAKE_TOOLCHAIN_FILE was set to C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg/scripts/buildsystems/vcpkg.cmake
C/C++: release|x86_64 :vcpkg_android.cmake: VCPKG_CHAINLOAD_TOOLCHAIN_FILE was set to C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393/build/cmake/android.toolchain.cmake
C/C++: release|x86_64 :CMake Error at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/scripts/buildsystems/vcpkg.cmake:883 (message):
C/C++: release|x86_64 :  vcpkg install failed.  See logs for more information:
C/C++: release|x86_64 :  C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\x86_64\vcpkg-manifest-install.log
C/C++: release|x86_64 :Call Stack (most recent call first):
C/C++: release|x86_64 :  C:/Users/erika/AppData/Local/Android/Sdk/cmake/3.22.1/share/cmake-3.22/Modules/CMakeDetermineSystem.cmake:124 (include)
C/C++: release|x86_64 :  CMakeLists.txt:32 (project)

> Task :app:configureCMakeRelWithDebInfo[x86_64] FAILED
> Task :app:mergeReleaseJniLibFolders UP-TO-DATE
> Task :app:checkReleaseDuplicateClasses UP-TO-DATE
> Task :app:dexBuilderRelease UP-TO-DATE
> Task :app:desugarReleaseFileDependencies UP-TO-DATE
> Task :app:mergeExtDexRelease UP-TO-DATE
> Task :app:mergeDexRelease UP-TO-DATE
> Task :app:mergeReleaseArtProfile UP-TO-DATE
> Task :app:compileReleaseArtProfile UP-TO-DATE
> Task :app:mergeReleaseShaders UP-TO-DATE
> Task :app:compileReleaseShaders NO-SOURCE
> Task :app:generateReleaseAssets UP-TO-DATE
> Task :app:mergeReleaseAssets UP-TO-DATE
> Task :app:compressReleaseAssets UP-TO-DATE
> Task :app:processReleaseJavaRes NO-SOURCE
> Task :app:mergeReleaseJavaResource UP-TO-DATE
> Task :app:optimizeReleaseResources UP-TO-DATE
> Task :app:collectReleaseDependencies UP-TO-DATE
> Task :app:sdkReleaseDependencyData UP-TO-DATE
> Task :app:writeReleaseAppMetadata UP-TO-DATE
> Task :app:writeReleaseSigningConfigVersions UP-TO-DATE

FAILURE: Build completed with 4 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:configureCMakeRelWithDebInfo[arm64-v8a]'.
> [CXX1405] error when building with cmake using C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\CMakeLists.txt: Build command failed.
  Error while executing process C:\Users\erika\AppData\Local\Android\Sdk\cmake\3.22.1\bin\cmake.exe with arguments {-HC:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client -DCMAKE_SYSTEM_NAME=Android -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_SYSTEM_VERSION=21 -DANDROID_PLATFORM=android-21 -DANDROID_ABI=arm64-v8a -DCMAKE_ANDROID_ARCH_ABI=arm64-v8a -DANDROID_NDK=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393 -DCMAKE_ANDROID_NDK=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393 -DCMAKE_TOOLCHAIN_FILE=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393\build\cmake\android.toolchain.cmake -DCMAKE_MAKE_PROGRAM=C:\Users\erika\AppData\Local\Android\Sdk\cmake\3.22.1\bin\ninja.exe -DCMAKE_CXX_FLAGS=-std=c++20 -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\build\intermediates\cxx\RelWithDebInfo\3e4p1i30\obj\arm64-v8a -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\build\intermediates\cxx\RelWithDebInfo\3e4p1i30\obj\arm64-v8a -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_FIND_ROOT_PATH=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\prefab\arm64-v8a\prefab -BC:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\arm64-v8a -GNinja -DVCPKG_TARGET_ANDROID=ON}
  -- Running vcpkg install
  Detecting compiler hash for triplet x64-windows...
  Detecting compiler hash for triplet arm64-android...
  The following packages will be built and installed:
      asio[core]:arm64-android -> 1.24.0#1
      liblzma[core]:arm64-android -> 5.4.0
      libogg[core]:arm64-android -> 1.3.5#1
      libvorbis[core]:arm64-android -> 1.3.7#2
      luajit[core]:arm64-android -> 2022-11-22
      nlohmann-json[core]:arm64-android -> 3.11.2
      openal-soft[core]:arm64-android -> 1.22.2#4
      openssl[core]:arm64-android -> 3.0.7#2
      parallel-hashmap[core]:arm64-android -> 1.3.8
      physfs[core]:arm64-android -> 3.2.0#1
    * protobuf[core]:x64-windows -> 3.21.8
      protobuf[core]:arm64-android -> 3.21.8
      stduuid[core]:arm64-android -> 1.2.2
    * vcpkg-cmake[core]:x64-windows -> 2022-11-13
    * vcpkg-cmake-config[core]:x64-windows -> 2022-02-06#1
      zlib[core]:arm64-android -> 1.2.13
  Additional packages (*) will be modified to complete this operation.
  Restored 6 package(s) from C:\Users\erika\AppData\Local\vcpkg\archives in 4.834 s. Use --debug to see more details.
  Installing 1/16 vcpkg-cmake-config:x64-windows...
  Elapsed time to handle vcpkg-cmake-config:x64-windows: 47.78 ms
  Installing 2/16 vcpkg-cmake:x64-windows...
  Elapsed time to handle vcpkg-cmake:x64-windows: 195.8 ms
  Installing 3/16 asio:arm64-android...
  Elapsed time to handle asio:arm64-android: 3.096 s
  Installing 4/16 liblzma:arm64-android...
  Elapsed time to handle liblzma:arm64-android: 180 ms
  Installing 5/16 libogg:arm64-android...
  Elapsed time to handle libogg:arm64-android: 80.6 ms
  Installing 6/16 libvorbis:arm64-android...
  Elapsed time to handle libvorbis:arm64-android: 110.4 ms
  Installing 7/16 luajit:arm64-android...
  Building luajit[core]:arm64-android...
  warning: -- Using community triplet arm64-android. This triplet configuration is not guaranteed to succeed.
  -- [COMMUNITY] Loading triplet configuration from: C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg\triplets\community\arm64-android.cmake
  -- Using cached LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz.
  -- Cleaning sources at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean. Use --editable to skip cleaning for the packages you specify.
  -- Extracting source C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/downloads/LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz
  -- Applying patch 003-do-not-set-macosx-deployment-target.patch
  -- Applying patch 004-fix-build-path-and-crt-linkage.patch
  -- Using source at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean
  -- Building arm64-android-dbg
  CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:58 (message):
    Failed to execute command "make -j5 -f
    C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean/Makefile
    clean" in working directory
    "C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean":
    The system cannot find the file specified
  Call Stack (most recent call first):
    ports/luajit/portfile.cmake:56 (vcpkg_execute_build_process)
    scripts/ports.cmake:147 (include)
  
  
  error: building luajit:arm64-android failed with: BUILD_FAILED
  error: Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
  Then check for known issues at:
      https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+luajit
  You can submit a new issue at:
      https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[luajit]+Build+error
  Include '[luajit] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
      vcpkg-tool version: 2022-11-10-5fdee72bc1fceca198fb1ab7589837206a8b81ba
      vcpkg-scripts version: 036769f5d 2023-01-11 (2 days ago)
  
  
  Please use the prefilled template from C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\arm64-v8a\vcpkg_installed\vcpkg\issue_body.md when reporting your issue.
  -- Running vcpkg install - failed
  -- Configuring incomplete, errors occurred!
  
  vcpkg_android.cmake: VCPKG_TARGET_TRIPLET was set to arm64-android
  vcpkg_android.cmake: CMAKE_TOOLCHAIN_FILE was set to C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg/scripts/buildsystems/vcpkg.cmake
  vcpkg_android.cmake: VCPKG_CHAINLOAD_TOOLCHAIN_FILE was set to C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393/build/cmake/android.toolchain.cmake
  CMake Error at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/scripts/buildsystems/vcpkg.cmake:883 (message):
    vcpkg install failed.  See logs for more information:
    C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\arm64-v8a\vcpkg-manifest-install.log
  Call Stack (most recent call first):
    C:/Users/erika/AppData/Local/Android/Sdk/cmake/3.22.1/share/cmake-3.22/Modules/CMakeDetermineSystem.cmake:124 (include)
    CMakeLists.txt:32 (project)
  
  


* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:configureCMakeRelWithDebInfo[armeabi-v7a]'.
> [CXX1405] error when building with cmake using C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\CMakeLists.txt: Build command failed.
  Error while executing process C:\Users\erika\AppData\Local\Android\Sdk\cmake\3.22.1\bin\cmake.exe with arguments {-HC:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client -DCMAKE_SYSTEM_NAME=Android -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_SYSTEM_VERSION=21 -DANDROID_PLATFORM=android-21 -DANDROID_ABI=armeabi-v7a -DCMAKE_ANDROID_ARCH_ABI=armeabi-v7a -DANDROID_NDK=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393 -DCMAKE_ANDROID_NDK=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393 -DCMAKE_TOOLCHAIN_FILE=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393\build\cmake\android.toolchain.cmake -DCMAKE_MAKE_PROGRAM=C:\Users\erika\AppData\Local\Android\Sdk\cmake\3.22.1\bin\ninja.exe -DCMAKE_CXX_FLAGS=-std=c++20 -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\build\intermediates\cxx\RelWithDebInfo\3e4p1i30\obj\armeabi-v7a -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\build\intermediates\cxx\RelWithDebInfo\3e4p1i30\obj\armeabi-v7a -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_FIND_ROOT_PATH=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\prefab\armeabi-v7a\prefab -BC:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\armeabi-v7a -GNinja -DVCPKG_TARGET_ANDROID=ON}
  -- Running vcpkg install
  Detecting compiler hash for triplet x64-windows...
  Detecting compiler hash for triplet arm-neon-android...
  The following packages will be built and installed:
      asio[core]:arm-neon-android -> 1.24.0#1
      liblzma[core]:arm-neon-android -> 5.4.0
      libogg[core]:arm-neon-android -> 1.3.5#1
      libvorbis[core]:arm-neon-android -> 1.3.7#2
      luajit[core]:arm-neon-android -> 2022-11-22
      nlohmann-json[core]:arm-neon-android -> 3.11.2
      openal-soft[core]:arm-neon-android -> 1.22.2#4
      openssl[core]:arm-neon-android -> 3.0.7#2
      parallel-hashmap[core]:arm-neon-android -> 1.3.8
      physfs[core]:arm-neon-android -> 3.2.0#1
    * protobuf[core]:x64-windows -> 3.21.8
      protobuf[core]:arm-neon-android -> 3.21.8
      stduuid[core]:arm-neon-android -> 1.2.2
    * vcpkg-cmake[core]:x64-windows -> 2022-11-13
    * vcpkg-cmake-config[core]:x64-windows -> 2022-02-06#1
      zlib[core]:arm-neon-android -> 1.2.13
  Additional packages (*) will be modified to complete this operation.
  Restored 6 package(s) from C:\Users\erika\AppData\Local\vcpkg\archives in 2.693 s. Use --debug to see more details.
  Installing 1/16 vcpkg-cmake-config:x64-windows...
  Elapsed time to handle vcpkg-cmake-config:x64-windows: 26.62 ms
  Installing 2/16 vcpkg-cmake:x64-windows...
  Elapsed time to handle vcpkg-cmake:x64-windows: 125.1 ms
  Installing 3/16 asio:arm-neon-android...
  Elapsed time to handle asio:arm-neon-android: 3.196 s
  Installing 4/16 liblzma:arm-neon-android...
  Elapsed time to handle liblzma:arm-neon-android: 130.6 ms
  Installing 5/16 libogg:arm-neon-android...
  Elapsed time to handle libogg:arm-neon-android: 76.37 ms
  Installing 6/16 libvorbis:arm-neon-android...
  Elapsed time to handle libvorbis:arm-neon-android: 111.2 ms
  Installing 7/16 luajit:arm-neon-android...
  Building luajit[core]:arm-neon-android...
  warning: -- Using community triplet arm-neon-android. This triplet configuration is not guaranteed to succeed.
  -- [COMMUNITY] Loading triplet configuration from: C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg\triplets\community\arm-neon-android.cmake
  -- Using cached LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz.
  -- Cleaning sources at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean. Use --editable to skip cleaning for the packages you specify.
  -- Extracting source C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/downloads/LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz
  -- Applying patch 003-do-not-set-macosx-deployment-target.patch
  -- Applying patch 004-fix-build-path-and-crt-linkage.patch
  -- Using source at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean
  -- Building arm-neon-android-dbg
  CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:58 (message):
    Failed to execute command "make -j5 -f
    C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean/Makefile
    clean" in working directory
    "C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean":
    The system cannot find the file specified
  Call Stack (most recent call first):
    ports/luajit/portfile.cmake:56 (vcpkg_execute_build_process)
    scripts/ports.cmake:147 (include)
  
  
  error: building luajit:arm-neon-android failed with: BUILD_FAILED
  error: Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
  Then check for known issues at:
      https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+luajit
  You can submit a new issue at:
      https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[luajit]+Build+error
  Include '[luajit] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
      vcpkg-tool version: 2022-11-10-5fdee72bc1fceca198fb1ab7589837206a8b81ba
      vcpkg-scripts version: 036769f5d 2023-01-11 (2 days ago)
  
  
  Please use the prefilled template from C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\armeabi-v7a\vcpkg_installed\vcpkg\issue_body.md when reporting your issue.
  -- Running vcpkg install - failed
  -- Configuring incomplete, errors occurred!
  
  vcpkg_android.cmake: VCPKG_TARGET_TRIPLET was set to arm-neon-android
  vcpkg_android.cmake: CMAKE_TOOLCHAIN_FILE was set to C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg/scripts/buildsystems/vcpkg.cmake
  vcpkg_android.cmake: VCPKG_CHAINLOAD_TOOLCHAIN_FILE was set to C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393/build/cmake/android.toolchain.cmake
  CMake Error at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/scripts/buildsystems/vcpkg.cmake:883 (message):
    vcpkg install failed.  See logs for more information:
    C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\armeabi-v7a\vcpkg-manifest-install.log
  Call Stack (most recent call first):
    C:/Users/erika/AppData/Local/Android/Sdk/cmake/3.22.1/share/cmake-3.22/Modules/CMakeDetermineSystem.cmake:124 (include)
    CMakeLists.txt:32 (project)
  
  


* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

3: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:configureCMakeRelWithDebInfo[x86]'.
> [CXX1405] error when building with cmake using C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\CMakeLists.txt: Build command failed.
  Error while executing process C:\Users\erika\AppData\Local\Android\Sdk\cmake\3.22.1\bin\cmake.exe with arguments {-HC:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client -DCMAKE_SYSTEM_NAME=Android -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_SYSTEM_VERSION=21 -DANDROID_PLATFORM=android-21 -DANDROID_ABI=x86 -DCMAKE_ANDROID_ARCH_ABI=x86 -DANDROID_NDK=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393 -DCMAKE_ANDROID_NDK=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393 -DCMAKE_TOOLCHAIN_FILE=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393\build\cmake\android.toolchain.cmake -DCMAKE_MAKE_PROGRAM=C:\Users\erika\AppData\Local\Android\Sdk\cmake\3.22.1\bin\ninja.exe -DCMAKE_CXX_FLAGS=-std=c++20 -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\build\intermediates\cxx\RelWithDebInfo\3e4p1i30\obj\x86 -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\build\intermediates\cxx\RelWithDebInfo\3e4p1i30\obj\x86 -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_FIND_ROOT_PATH=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\prefab\x86\prefab -BC:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\x86 -GNinja -DVCPKG_TARGET_ANDROID=ON}
  -- Running vcpkg install
  Detecting compiler hash for triplet x64-windows...
  Detecting compiler hash for triplet x86-android...
  The following packages will be built and installed:
      asio[core]:x86-android -> 1.24.0#1
      liblzma[core]:x86-android -> 5.4.0
      libogg[core]:x86-android -> 1.3.5#1
      libvorbis[core]:x86-android -> 1.3.7#2
      luajit[core]:x86-android -> 2022-11-22
      nlohmann-json[core]:x86-android -> 3.11.2
      openal-soft[core]:x86-android -> 1.22.2#4
      openssl[core]:x86-android -> 3.0.7#2
      parallel-hashmap[core]:x86-android -> 1.3.8
      physfs[core]:x86-android -> 3.2.0#1
    * protobuf[core]:x64-windows -> 3.21.8
      protobuf[core]:x86-android -> 3.21.8
      stduuid[core]:x86-android -> 1.2.2
    * vcpkg-cmake[core]:x64-windows -> 2022-11-13
    * vcpkg-cmake-config[core]:x64-windows -> 2022-02-06#1
      zlib[core]:x86-android -> 1.2.13
  Additional packages (*) will be modified to complete this operation.
  Restored 6 package(s) from C:\Users\erika\AppData\Local\vcpkg\archives in 2.334 s. Use --debug to see more details.
  Installing 1/16 vcpkg-cmake-config:x64-windows...
  Elapsed time to handle vcpkg-cmake-config:x64-windows: 37.59 ms
  Installing 2/16 vcpkg-cmake:x64-windows...
  Elapsed time to handle vcpkg-cmake:x64-windows: 130 ms
  Installing 3/16 asio:x86-android...
  Elapsed time to handle asio:x86-android: 2.501 s
  Installing 4/16 liblzma:x86-android...
  Elapsed time to handle liblzma:x86-android: 140.7 ms
  Installing 5/16 libogg:x86-android...
  Elapsed time to handle libogg:x86-android: 78.21 ms
  Installing 6/16 libvorbis:x86-android...
  Elapsed time to handle libvorbis:x86-android: 110.3 ms
  Installing 7/16 luajit:x86-android...
  Building luajit[core]:x86-android...
  warning: -- Using community triplet x86-android. This triplet configuration is not guaranteed to succeed.
  -- [COMMUNITY] Loading triplet configuration from: C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg\triplets\community\x86-android.cmake
  -- Using cached LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz.
  -- Cleaning sources at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean. Use --editable to skip cleaning for the packages you specify.
  -- Extracting source C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/downloads/LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz
  -- Applying patch 003-do-not-set-macosx-deployment-target.patch
  -- Applying patch 004-fix-build-path-and-crt-linkage.patch
  -- Using source at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean
  -- Building x86-android-dbg
  CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:58 (message):
    Failed to execute command "make -j5 -f
    C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean/Makefile
    clean" in working directory
    "C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean":
    The system cannot find the file specified
  Call Stack (most recent call first):
    ports/luajit/portfile.cmake:56 (vcpkg_execute_build_process)
    scripts/ports.cmake:147 (include)
  
  
  error: building luajit:x86-android failed with: BUILD_FAILED
  error: Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
  Then check for known issues at:
      https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+luajit
  You can submit a new issue at:
      https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[luajit]+Build+error
  Include '[luajit] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
      vcpkg-tool version: 2022-11-10-5fdee72bc1fceca198fb1ab7589837206a8b81ba
      vcpkg-scripts version: 036769f5d 2023-01-11 (2 days ago)
  
  
  Please use the prefilled template from C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\x86\vcpkg_installed\vcpkg\issue_body.md when reporting your issue.
  -- Running vcpkg install - failed
  -- Configuring incomplete, errors occurred!
  
  vcpkg_android.cmake: VCPKG_TARGET_TRIPLET was set to x86-android
  vcpkg_android.cmake: CMAKE_TOOLCHAIN_FILE was set to C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg/scripts/buildsystems/vcpkg.cmake
  vcpkg_android.cmake: VCPKG_CHAINLOAD_TOOLCHAIN_FILE was set to C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393/build/cmake/android.toolchain.cmake
  CMake Error at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/scripts/buildsystems/vcpkg.cmake:883 (message):
    vcpkg install failed.  See logs for more information:
    C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\x86\vcpkg-manifest-install.log
  Call Stack (most recent call first):
    C:/Users/erika/AppData/Local/Android/Sdk/cmake/3.22.1/share/cmake-3.22/Modules/CMakeDetermineSystem.cmake:124 (include)
    CMakeLists.txt:32 (project)
  
  


* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

4: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:configureCMakeRelWithDebInfo[x86_64]'.
> [CXX1405] error when building with cmake using C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\CMakeLists.txt: Build command failed.
  Error while executing process C:\Users\erika\AppData\Local\Android\Sdk\cmake\3.22.1\bin\cmake.exe with arguments {-HC:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client -DCMAKE_SYSTEM_NAME=Android -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_SYSTEM_VERSION=21 -DANDROID_PLATFORM=android-21 -DANDROID_ABI=x86_64 -DCMAKE_ANDROID_ARCH_ABI=x86_64 -DANDROID_NDK=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393 -DCMAKE_ANDROID_NDK=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393 -DCMAKE_TOOLCHAIN_FILE=C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393\build\cmake\android.toolchain.cmake -DCMAKE_MAKE_PROGRAM=C:\Users\erika\AppData\Local\Android\Sdk\cmake\3.22.1\bin\ninja.exe -DCMAKE_CXX_FLAGS=-std=c++20 -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\build\intermediates\cxx\RelWithDebInfo\3e4p1i30\obj\x86_64 -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\build\intermediates\cxx\RelWithDebInfo\3e4p1i30\obj\x86_64 -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_FIND_ROOT_PATH=C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\prefab\x86_64\prefab -BC:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\x86_64 -GNinja -DVCPKG_TARGET_ANDROID=ON}
  -- Running vcpkg install
  Detecting compiler hash for triplet x64-windows...
  Detecting compiler hash for triplet x64-android...
  The following packages will be built and installed:
      asio[core]:x64-android -> 1.24.0#1
      liblzma[core]:x64-android -> 5.4.0
      libogg[core]:x64-android -> 1.3.5#1
      libvorbis[core]:x64-android -> 1.3.7#2
      luajit[core]:x64-android -> 2022-11-22
      nlohmann-json[core]:x64-android -> 3.11.2
      openal-soft[core]:x64-android -> 1.22.2#4
      openssl[core]:x64-android -> 3.0.7#2
      parallel-hashmap[core]:x64-android -> 1.3.8
      physfs[core]:x64-android -> 3.2.0#1
    * protobuf[core]:x64-windows -> 3.21.8
      protobuf[core]:x64-android -> 3.21.8
      stduuid[core]:x64-android -> 1.2.2
    * vcpkg-cmake[core]:x64-windows -> 2022-11-13
    * vcpkg-cmake-config[core]:x64-windows -> 2022-02-06#1
      zlib[core]:x64-android -> 1.2.13
  Additional packages (*) will be modified to complete this operation.
  Restored 6 package(s) from C:\Users\erika\AppData\Local\vcpkg\archives in 2.671 s. Use --debug to see more details.
  Installing 1/16 vcpkg-cmake-config:x64-windows...
  Elapsed time to handle vcpkg-cmake-config:x64-windows: 26.32 ms
  Installing 2/16 vcpkg-cmake:x64-windows...
  Elapsed time to handle vcpkg-cmake:x64-windows: 129.9 ms
  Installing 3/16 asio:x64-android...
  Elapsed time to handle asio:x64-android: 2.598 s
  Installing 4/16 liblzma:x64-android...
  Elapsed time to handle liblzma:x64-android: 135.3 ms
  Installing 5/16 libogg:x64-android...
  Elapsed time to handle libogg:x64-android: 76.93 ms
  Installing 6/16 libvorbis:x64-android...
  Elapsed time to handle libvorbis:x64-android: 114.9 ms
  Installing 7/16 luajit:x64-android...
  Building luajit[core]:x64-android...
  warning: -- Using community triplet x64-android. This triplet configuration is not guaranteed to succeed.
  -- [COMMUNITY] Loading triplet configuration from: C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg\triplets\community\x64-android.cmake
  -- Using cached LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz.
  -- Cleaning sources at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean. Use --editable to skip cleaning for the packages you specify.
  -- Extracting source C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/downloads/LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz
  -- Applying patch 003-do-not-set-macosx-deployment-target.patch
  -- Applying patch 004-fix-build-path-and-crt-linkage.patch
  -- Using source at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean
  -- Building x64-android-dbg
  CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:58 (message):
    Failed to execute command "make -j5 -f
    C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean/Makefile
    clean" in working directory
    "C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean":
    The system cannot find the file specified
  Call Stack (most recent call first):
    ports/luajit/portfile.cmake:56 (vcpkg_execute_build_process)
    scripts/ports.cmake:147 (include)
  
  
  error: building luajit:x64-android failed with: BUILD_FAILED
  error: Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
  Then check for known issues at:
      https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+luajit
  You can submit a new issue at:
      https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[luajit]+Build+error
  Include '[luajit] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
      vcpkg-tool version: 2022-11-10-5fdee72bc1fceca198fb1ab7589837206a8b81ba
      vcpkg-scripts version: 036769f5d 2023-01-11 (2 days ago)
  
  
  Please use the prefilled template from C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\x86_64\vcpkg_installed\vcpkg\issue_body.md when reporting your issue.
  -- Running vcpkg install - failed
  -- Configuring incomplete, errors occurred!
  
  vcpkg_android.cmake: VCPKG_TARGET_TRIPLET was set to x64-android
  vcpkg_android.cmake: CMAKE_TOOLCHAIN_FILE was set to C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg/scripts/buildsystems/vcpkg.cmake
  vcpkg_android.cmake: VCPKG_CHAINLOAD_TOOLCHAIN_FILE was set to C:\Users\erika\AppData\Local\Android\Sdk\ndk\25.1.8937393/build/cmake/android.toolchain.cmake
  CMake Error at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/scripts/buildsystems/vcpkg.cmake:883 (message):
    vcpkg install failed.  See logs for more information:
    C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\x86_64\vcpkg-manifest-install.log
  Call Stack (most recent call first):
    C:/Users/erika/AppData/Local/Android/Sdk/cmake/3.22.1/share/cmake-3.22/Modules/CMakeDetermineSystem.cmake:124 (include)
    CMakeLists.txt:32 (project)
  
  


* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 46s
45 actionable tasks: 5 executed, 40 up-to-date
</details>

The file from vcpkg-manifest-install.log has actually the same log as in the build log:
<details>
  <summary>vcpkg-manifest-install.log</summary>
Detecting compiler hash for triplet x64-windows...
Detecting compiler hash for triplet arm64-android...
The following packages will be built and installed:
    asio[core]:arm64-android -> 1.24.0#1
    liblzma[core]:arm64-android -> 5.4.0
    libogg[core]:arm64-android -> 1.3.5#1
    libvorbis[core]:arm64-android -> 1.3.7#2
    luajit[core]:arm64-android -> 2022-11-22
    nlohmann-json[core]:arm64-android -> 3.11.2
    openal-soft[core]:arm64-android -> 1.22.2#4
    openssl[core]:arm64-android -> 3.0.7#2
    parallel-hashmap[core]:arm64-android -> 1.3.8
    physfs[core]:arm64-android -> 3.2.0#1
  * protobuf[core]:x64-windows -> 3.21.8
    protobuf[core]:arm64-android -> 3.21.8
    stduuid[core]:arm64-android -> 1.2.2
  * vcpkg-cmake[core]:x64-windows -> 2022-11-13
  * vcpkg-cmake-config[core]:x64-windows -> 2022-02-06#1
    zlib[core]:arm64-android -> 1.2.13
Additional packages (*) will be modified to complete this operation.
Restored 6 package(s) from C:\Users\erika\AppData\Local\vcpkg\archives in 4.834 s. Use --debug to see more details.
Installing 1/16 vcpkg-cmake-config:x64-windows...
Elapsed time to handle vcpkg-cmake-config:x64-windows: 47.78 ms
Installing 2/16 vcpkg-cmake:x64-windows...
Elapsed time to handle vcpkg-cmake:x64-windows: 195.8 ms
Installing 3/16 asio:arm64-android...
Elapsed time to handle asio:arm64-android: 3.096 s
Installing 4/16 liblzma:arm64-android...
Elapsed time to handle liblzma:arm64-android: 180 ms
Installing 5/16 libogg:arm64-android...
Elapsed time to handle libogg:arm64-android: 80.6 ms
Installing 6/16 libvorbis:arm64-android...
Elapsed time to handle libvorbis:arm64-android: 110.4 ms
Installing 7/16 luajit:arm64-android...
Building luajit[core]:arm64-android...
warning: -- Using community triplet arm64-android. This triplet configuration is not guaranteed to succeed.
-- [COMMUNITY] Loading triplet configuration from: C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg\triplets\community\arm64-android.cmake
-- Using cached LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz.
-- Cleaning sources at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/downloads/LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz
-- Applying patch 003-do-not-set-macosx-deployment-target.patch
-- Applying patch 004-fix-build-path-and-crt-linkage.patch
-- Using source at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean
-- Building arm64-android-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:58 (message):
  Failed to execute command "make -j5 -f
  C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean/Makefile
  clean" in working directory
  "C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean":
  The system cannot find the file specified
Call Stack (most recent call first):
  ports/luajit/portfile.cmake:56 (vcpkg_execute_build_process)
  scripts/ports.cmake:147 (include)


error: building luajit:arm64-android failed with: BUILD_FAILED
error: Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+luajit
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[luajit]+Build+error
Include '[luajit] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
    vcpkg-tool version: 2022-11-10-5fdee72bc1fceca198fb1ab7589837206a8b81ba
    vcpkg-scripts version: 036769f5d 2023-01-11 (2 days ago)


Please use the prefilled template from C:\Users\erika\source\repos\ErikasKontenis\onlyfortress-client\android\app\.cxx\RelWithDebInfo\3e4p1i30\arm64-v8a\vcpkg_installed\vcpkg\issue_body.md when reporting your issue.

</details>

As you can see from my vcpkg version (it is several days old) I have recently did git pull & vcpkg update however it didn't help me.

My main question would be do I have some kind of workaround to compile this? Maybe you could share with me the LuaJIT version you have and which does build. And does it fails to build just for me or it is everyone impacted. 

Furthermore, I have also tried to simply open the cmd in the vcpkg folder and run the command: 
`vcpkg install luajit --triplet=x64-android`

I do get the same issue about LuaJIT, HOWEVER the LuaJIT builds fine for windows build and does not build for all android triplets.

<details>
  <summary>vcpkg install luajit --triplet=x64-android</summary>
C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg>vcpkg install luajit --triplet=x64-android
Computing installation plan...
The following packages will be built and installed:
    luajit[core]:x64-android -> 2022-11-22
Detecting compiler hash for triplet x64-android...
Restored 0 package(s) from C:\Users\erika\AppData\Local\vcpkg\archives in 271.2 us. Use --debug to see more details.
Installing 1/1 luajit:x64-android...
Building luajit[core]:x64-android...
warning: -- Using community triplet x64-android. This triplet configuration is not guaranteed to succeed.
-- [COMMUNITY] Loading triplet configuration from: C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg\triplets\community\x64-android.cmake
-- Downloading https://github.com/LuaJIT/LuaJIT/archive/46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz -> LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz...
-- Extracting source C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/downloads/LuaJIT-LuaJIT-46aa45dcbd9f3843503ddf3e00f8dda89eac6789.tar.gz
-- Applying patch 003-do-not-set-macosx-deployment-target.patch
-- Applying patch 004-fix-build-path-and-crt-linkage.patch
-- Using source at C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean
-- Building x64-android-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:58 (message):
  Failed to execute command "make -j5 -f
  C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean/Makefile
  clean" in working directory
  "C:/Users/erika/source/repos/ErikasKontenis/onlyfortress/vcpkg/buildtrees/luajit/src/a89eac6789-b23d9d3b47.clean":
  The system cannot find the file specified
Call Stack (most recent call first):
  ports/luajit/portfile.cmake:56 (vcpkg_execute_build_process)
  scripts/ports.cmake:147 (include)


error: building luajit:x64-android failed with: BUILD_FAILED
error: Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+luajit
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[luajit]+Build+error
Include '[luajit] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
    vcpkg-tool version: 2022-11-10-5fdee72bc1fceca198fb1ab7589837206a8b81ba
    vcpkg-scripts version: 036769f5d 2023-01-11 (2 days ago)


Please use the prefilled template from C:\Users\erika\source\repos\ErikasKontenis\onlyfortress\vcpkg\installed\vcpkg\issue_body.md when reporting your issue.
</details>

**Comment by tuliomagalhaes** [comment]
I noticed that our CMake is really broken for Android, I opened a PR, however we have other fixes that have to be done before trying to build for Android, I've described it [here](https://github.com/mehah/otclient/pull/387#issuecomment-1382836653). You have to download the libraries again as now it includes the luajit include folder.

**Comment by ErikasKontenis** [comment]
Oh now I see that LuaJIT isn't even supposed to be loaded from vcpkg but taken from android libraries. Thank you and I am looking forward your PR and how it will end up with the other fixes.
